wrangles/recipe.py

```diff
@@ -16,6 +16,9 @@
 from . import recipe_wrangles as _recipe_wrangles
 from . import connectors as _connectors
 from .config import no_where_list
+from wrangles.data import model_content as _model_content
```

Comment:
You should use the "from . import x as y" style import. The wrangles.data style can have issues when it's a package, depending on the circumstances.